Zilliqa

```diff
@@ -0,0 +1,5 @@
+Blockchain Software Engineer (C++)
+Compiler Engineer
+Full-Stack Developer
+Blockchain Application Engineer
+Blockchain Researcher
```

Comment:
**As a Blockchain Researcher, you will:**

- Conduct cutting-edge research to solve the pressing challenges for Zilliqa and blockchains in general.
- Work closely with other team members to develop research outcomes into product-level implementation and verify assumptions and hypotheses with experiments.
- Writing and publishing documents and papers to detail the research findings and experimental results.
- Work closely with collaborators in the community or partner enterprises.

**Qualifications**

- PhD degree in computer science, computer engineering, or equivalent practical experience.
- Excellent track record of publications in international conferences and journals in one or more of the following domains: cybersecurity, blockchains, distributed systems and storage, applied cryptography, privacy preserving (or enhancing) technologies, computer networks, formal methods, programming languages.
- Ability to work independently and collaboratively in an agile team.

https://zilliqa.com/careers/blockchain-researcher.html